internal/pkg/config/env.go

```diff
@@ -0,0 +1,130 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+// Environment represents the configuration of a particular environment in a App. It includes
+// the environment's account and region, name, as well as the App it belongs to.
+type Environment struct {
```

Comment:
What do you think of just:
```suggestion
// Environment represents a deployment environment in an application. 
```

Reply:
<3